docs/api-inference/tasks/translation.md

```diff
@@ -0,0 +1,110 @@
+## Translation
+
+Translation is the task of converting text from one language to another.
+
+<Tip>
+
+For more details about the `translation` task, check out its [dedicated page](https://huggingface.co/tasks/translation)! You will find examples and related materials.
+
+</Tip>
+
+### Recommended models
+
+- [Helsinki-NLP/opus-mt-en-fr](https://huggingface.co/Helsinki-NLP/opus-mt-en-fr): A model that translates from English to French.
+- [t5-base](https://huggingface.co/t5-base): A general-purpose Transformer that can be used to translate from English to German, French, or Romanian.
+
+This is only a subset of the supported models. Find the model that suits you best [here](https://huggingface.co/models?inference=warm&pipeline_tag=translation&sort=trending).
+
+### API specification
+
+#### Request
+
+| Payload |  |  |
+| :--- | :--- | :--- |
+| **inputs*** | _string_ | The text to translate. |
+| **parameters** | _object_ | Additional inference parameters for Translation |
+| **&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;src_lang** | _string_ | The source language of the text. Required for models that can translate from multiple languages. |
+| **&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;tgt_lang** | _string_ | Target language to translate to. Required for models that can translate to multiple languages. |
```

Comment:
Where do these come from? I'm looking at https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/tasks/translation/inference.ts#L20 and https://github.com/huggingface/huggingface.js/blob/main/packages/tasks/src/tasks/translation/spec/input.json

Reply:
I opened a PR in `@huggingface.js/tasks` to add modifications to the existing specs (see https://github.com/huggingface/huggingface.js/pull/872). And [here](https://github.com/huggingface/hub-docs/blob/4039c7eb276f14139e86d851259074e9631ad11e/scripts/api-inference/scripts/generate.ts#L18) in the `generate.ts` script I make sure to pull the specs from the `"update-specification-for-docs"` revision. We will have to switch to main once all specs are good.